plugins/StereoControl/StereoControl.cpp

```diff
@@ -281,11 +281,11 @@ bool StereoControlEffect::processAudioBuffer(sampleFrame* buf, const fpp_t frame
 
 				if (pan >= 0)
 				{
-					haasDelayVal[0] = linearInterpolate(0, 0.0008f * m_sampleRate * panDelay, abs(pan));
+					haasDelayVal[0] = 0.0008f * m_sampleRate * panDelay * asin(pan);
```

Comment:
The delay value should increase slower than linear, but it's faster than linear.